core/dbt/task/runnable.py

```diff
@@ -406,11 +406,21 @@ def populate_adapter_cache(
         if not self.args.populate_cache:
             return
 
+        if self.manifest is None:
+            raise DbtInternalError("manifest was None in populate_adapter_cache")
```

Comment:
Any idea under what conditions we wouldn't have a manifest?

Reply:
This was primarily added out of defensiveness &  mypy appeasement , and is not an real expected condition -- it would be our own improper usage of populate_adapter_cache. Since GraphRunnableTask inherits from ConfiguredTask which has an optional manifest, mypy was flagging the None-value of manifest as potentially problematic. the manifest gets set earlier on during GraphRunnableTask execution and should not be None by this point.